avocado_vt/test.py

```diff
@@ -256,16 +257,16 @@ def setUp(self):
         except exceptions.TestSkipError:
             raise   # This one has to be raised in setUp
```

Comment:
Nice catch, shouldn't you set the `self._exc_info` here as well? (I know the TestSkipError is not used in the finally block, but it might be in the future and saving it would be IMO more consistent).